### PR TITLE
config(tidb): update needs cherry pick labels

### PR DESCRIPTION
**Pull request by wuhuizuo**
for versions label from `needs-cherry-pick-to-{ .ver }` to `needs-cherry-pick-to-release-{ .ver }`:
 - `6.1`
 - `6.3`
 - `5.3`
 - `5.4`

### What problem does this PR solve?

Issue Number: close #xxx 

Problem Summary:

### What is changed and how it works?

What's Changed:

How it Works:

### CODE REVIEW

**Comment by MimeLyc** [comment]
lgtm

**Review by Mini256 — APPROVED** [review]
rest LGTM

File: prow/config/labels.yaml [hunk 1]
```diff
@@ -4276,7 +4276,7 @@ repos:
       - name: needs-cherry-pick-6.3
         color: "ededed"
         target: both
-      # - name: needs-cherry-pick-{{.ver}}
+      # - name: needs-cherry-pick-release-{{.ver}}
```
[the remark this answers is not in the document]
Comment (wuhuizuo) [reply prow/config/labels.yaml:4279]: it's a guide for new label.

**Pull request merged**